### PR TITLE
Add creat time on detail page (fixes #350).

### LINKED ISSUE #350: Display the created date & time on the work items
As part of the "Date / Timestamp on new Work Items " story: https://github.com/almighty/almighty-core/issues/499 the UI should display the created date & time on the work items

### PR DESCRIPTION
**Pull request by vikram-raj**
This PR requires you to run 'npm i'.
Fixes #350

### CODE REVIEW

**Comment by aslakknutsen** [comment]
[test]

File: src/app/pipes/alm-moment-time.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import * as moment from 'moment';
+
+/*
```
Comment (sanbornsen) [inline src/app/pipes/alm-moment-time.pipe.ts]: Nice gesture of comments.

**Comment by aslakknutsen** [comment]
We could use AlmMomentTime for comments/created-at as well

**Comment by vikram-raj** [comment]
@aslakknutsen Yes, we can use it.

**Comment by joshuawilson** [comment]
[test]

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html [hunk 2]
```diff
@@ -87,7 +87,7 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
           </div>
         </div>
       </div>
-      <div class="col-md-12 col-sm-12 col-xs-12 detail-assignee-wrap">
+      <div class="col-md-12 col-sm-12 col-xs-12 detail-assignee-wrap">
```
Comment (joshuawilson) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html]: why was a space added to the end of this line?

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.module.ts [hunk 3]
```diff
@@ -29,8 +30,20 @@ import { WorkItemCommentService } from './work-item-comment/work-item-comment.se
     FormsModule,
     Ng2CompleterModule
   ],
-  declarations: [WorkItemDetailComponent, AlmTrim, AlmSearchHighlight, AlmLinkTarget, AlmAvatarSize, WorkItemLinkComponent, WorkItemCommentComponent],
+  declarations: [
+    WorkItemDetailComponent,
```
Comment (joshuawilson) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.module.ts]: This should be alphabetical.

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.module.ts [hunk 4]
```diff
@@ -12,6 +12,7 @@ import { AlmTrim } from '../../../pipes/alm-trim';
 import { AlmSearchHighlight } from '../../../pipes/alm-search-highlight.pipe';
 import { AlmLinkTarget } from '../../../pipes/alm-link-target.pipe';
 import { AlmAvatarSize } from '../../../pipes/alm-avatar-size.pipe';
+import { AlmMomentTime } from '../../../pipes/alm-moment-time.pipe';
```
Comment (joshuawilson) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.module.ts]: This should be alphabetical.

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.spec.ts [hunk 5]
```diff
@@ -242,7 +243,8 @@ describe('Detailed view and edit a selected work item - ', () => {
         AlmTrim,
         AlmSearchHighlight,
         AlmLinkTarget,
-        AlmAvatarSize
+        AlmAvatarSize,
```
Comment (joshuawilson) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.spec.ts]: This should be alphabetical.

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**